fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts

```diff
@@ -1594,9 +1597,11 @@ export class PseudoFWC {
                     }
 
                     if (value.failure === 3) {
+                        console.log(`Playing Master warning Source:${key}`);
```

Comment:
Pls remove the logs and we're good to go :)

Reply:
duh, knew I forgot smth